src/development/sic-calls/history.md

```diff
@@ -13,6 +14,93 @@
+
+[@ignaciohagopian](https://x.com/ignaciohagopian) explains that geth works today by automatically recording the MPT preimages when it ingests the provided fixtures pre-state. He believes we shouldn’t blow up the testing framework with extra fields if EL clients can derive it from existing data.
+
+@spencer from the testing team made it even clearer that you can interpret the preimages from the pre-state, which should be enough compared to a more explicit field of preimages. In summary, we won’t add the preimages as a new field in fixtures, and clients should continue to record them as they ingest the provided fixture pre-state.
+
+### 5. State Expiry discussion
+
+[@g11tech](https://x.com/g11tech) raised a topic outside the agenda that is still related to the tree conversion topics. Can we reuse the code of the transition to implement state expiry?
+
+[@sophia](https://x.com/_sophiagold_) provided more historical context and confirmed that trying to achieve expiry by using one tree per epoch was very complex since it requires introducing address space extensions and has high UX friction. She provided the following links, which dive deeper into this topic: a potentially [underexplored idea](https://ethereum-magicians.org/t/types-of-resurrection-metadata-in-state-expiry/6607) that could be a viable solution and a [summary from Vitalik from some time ago about state expiry](https://hackmd.io/@vbuterin/state_expiry_paths).
+
+[@ignaciohagopian](https://x.com/ignaciohagopian) mentioned that we should try allocating this topic to a future SIC agenda so we have time to continue the discussion if needed.
+
+### 6. Post-quantum Verkle Trees variants
+
+[@CPerezz19](https://x.com/CPerezz19) [wrote a document](https://hackmd.io/qeEJsgjVSDeMjiR8d3-UvQ?both) that summarizes the main benefits of the current Verkle Tree design and how it compares with other presented alternatives. In particular, it explores potential variants that address the post-quantum weakness of the current design.
+
+He has been talking with many experts in the field and found out there’s active interest in the topic. This might signal an underexplored topic, meaning there might be viable PQ variants for a Verkle Tree design.
+
+The next step is to get more cryptography experts involved in the presented document to have feedback and potentially do some benchmarks to confirm if any of these variants could be a feasible contender to be proposed.
+
+Future SIC calls will continue to discuss this topic since we need to give time for more people to chime into Carlos's work and provide feedback.
+
+### 7. EXTCODECOPY behavior when targeting system contracts
+
+[@gballet](https://x.com/gballet) raised whether doing an EXTCODECOPY of a system contract should include system contract code chunks in the witness.
```

Comment:
`EXTCODECOPY` ?

Reply:
![image](https://github.com/user-attachments/assets/67e8a343-b5e3-47ce-8d65-2193f307c162)
Yes?

Reply:
I meant to code-style it